csrc/python_frontend/python_bindings.cpp

```diff
@@ -703,6 +703,11 @@ void initNvFuserPythonBindings(PyObject* module) {
             self.print(ss);
             return ss.str();
           })
+      .def(
+          "_clone",
+          [](FusionDefinition& self, FusionDefinition& other) {
+            self.clone(other);
```

Comment:
any reason for this indirection here? if we are just calling `self.clone` anyway.

Reply:
sorry this is just meant to be a nitpicking here.

If clone is indeed just a debug function, let's not put it inside FusionDefinition, but rather as a function directly here.

Reply:
I'm not sure what you mean. Can you add a code snippet example?

Reply:
Apology for vague comment.

So instead of having clone as a member function inside `FusionDefinition`. I think we should push this out as a function in `python_bindings.cpp`, if it's meant to be just a debug function.

```
void FusionDefinition::clone(FusionDefinition& other) {
  NVF_CHECK(completed(), "FusionDefinition definition does not exist!");
  NVF_ERROR(
      !other.completed(),
      "Expected an incomplete definition before translation.");
  translate(preschedFusion(), &other);
}
```

i.e.

```
void clone(FusionDefinition& from, FusionDefinition& to) {
  NVF_CHECK(from.completed(), "FusionDefinition definition does not exist!");
  NVF_ERROR(
      !to.completed(),
      "Expected an incomplete definition before translation.");
  // Something like this would work? vvv
  translate(fusionCache()->queryFusionSchedules(from.id())->preschedFusion(), &to);
}
```